src/stars.css

```diff
@@ -0,0 +1,91 @@
+.tutto {
+  height: 160px;
+  width: 140px;
+}
+
+.stars {
+  display: flex;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.stars--0 > :nth-child(n) {
+  background-image: url("./images/star.svg");
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--1 > :first-child {
+  background-image: url("./images/star-active.svg");
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--1 > :nth-child(n+2) {
+  background-image: url("./images/star.svg");
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--2 > :nth-child(n-2) {
+  background-image: url("./images/star-active.svg");
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--2 > :nth-child(n+3) {
+  background-image: url("./images/star.svg");
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+}
```

Comment:
group

Reply:
thanks.
in new version they are grouped